src/sequences/Formula.ts

```diff
@@ -30,24 +34,24 @@ class Formula extends Cached {
      * @param {*} sequenceID the sequence identifier of the sequence
      */
     constructor(sequenceID: number) {
-        super(sequenceID)
+        super(paramDesc, sequenceID)
         // It is mandatory to initialize the `evaluator` property here,
         // so just use a simple dummy formula until the user provides one.
         this.evaluator = math.compile(this.formula)
     }
 
-    checkParameters() {
-        const status = super.checkParameters()
+    checkParameters(params: ParamValues<typeof paramDesc>) {
+        const status = super.checkParameters(params)
+        params.formula
```

Comment:
Confused; does evaluating params.formula do something? Isn't params.formula just a string? This line looks like a useless evaluation of a variable; I am surprised there isn't a compiler or at least eslint warning, but there does not seem to be.

Reply:
There still seems to be an extraneous evaluation of params.formula here but in the interest of expediency I will try removing it myself.

Reply:
Actually, since there is a remaining bug you need to deal with anyway, please do remove this in your next commit. Thanks.